workspace/dataset/CostCenter_Dim_NonPII.JSON

```diff
@@ -14,64 +14,54 @@
         "fileSystem": "odw-harmonised"
```

Comment:
This file was corrupted ~ 3 years ago, it needed to be fixed as part of this PR for the new task to work